tests/unit/answers-headless.ts

```diff
@@ -66,7 +66,7 @@ const mockedCore: any = {
   filterSearch: jest.fn(() => Promise.resolve({}))
 };
 
-const answers = new AnswersHeadless(mockedCore, mockedStateManager, new HttpManager());
+const answers = new SearchHeadless(mockedCore, mockedStateManager, new HttpManager());
```

Comment:
will there be another part for updating the file names, variables and docs in tests folder as well?